hw12_13_14_15_calendar/service/schema/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Copypaste from https://github.com/grpc-ecosystem/grpc-gateway/tree/master/third_party/googleapis/google/api
to make it work